shepherd/manage.py

```diff
@@ -50,14 +48,6 @@ def run(host, port, config_file) -> None:
     api_server = WSGIServer((host, port), app, log=logging.getLogger(''))
     api_handler = gevent.spawn(api_server.serve_forever)
 
-    # everything should be ready, lets check if minio works
```

Comment:
Is there a solid reasoning behind removing this check? It was quite convenient as you would get immediate error if you forget to start the storage.

Reply:
It's not convenient at all. In a docker-compose deployment, you need to start the shepherd with a delay, which is a) slow (Minio usually won't need the whole second) b) unreliable (when docker-compose is e.g. pulling a new minio image, one second might not be enough) c) ugly. If you forget to start Minio now, you'll get a stern warning without a crash - that's not much of a difference for a developer who launches the shepherd in a terminal.

In short, it's OK for Minio to not be accessible when shepherd starts, but it's absolutely not OK when it crashes after some time (think disk space running out). Until now, we checked exactly the opposite conditions.

Reply:
Fair points.